randovania/games/samus_returns/logic_database/Area 4 - East.txt

```diff
@@ -370,7 +370,9 @@ Extra - asset_id: collision_camera_007
   * Extra - actor_name: LE_Item_005
   * Extra - actor_type: item_powerbombtank
   > Tunnel to Main Shaft
-      Grapple Beam and Spiderspark (Intermediate) and Can Spiderspark and Lay Power Bomb
+      Any of the following:
+          Grapple Beam or Lay Power Bomb
+          Spiderspark (Beginner) and Can Spiderspark
```

Comment:
Remove `Lay Power Bomb` since it is now redundant.

Reply:
Its been removed. Thanks for looking over these, i'll continue to see if there's anything else I can add.